api/RTCIceCandidate.json

```diff
@@ -151,10 +151,12 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "22",
+              "notes": "Before Firefox 126, the property was not read-only."
```

Comment:
FYI @Elchi3 FF126 made [candidate](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/candidate), [sdpMid](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/sdpMid), [sdpMLineIndex](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/sdpMLineIndex), [usernameFragment](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate/usernameFragment) match the spec by changing them to "read-only". I've added this note in the appropriate places in commit https://github.com/mdn/browser-compat-data/pull/22854/commits/ea0317e5c8f1947d784a513d659623f2c99426f4